src/containers/Profiles/UserProfileContainer.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { fetchUserProfile } from '../../store/actions/async/profiles';
+import { fetchArticlesAction } from '../../store/actions/async/ArticleActions';
+import UserProfile from '../../components/Profiles/UserProfile';
+import { dismissMessage } from '../../store/actions/sync/profiles';
+import PropTypes from 'prop-types';
+
+export class UserProfileContainer extends Component {
+    fetchProfile = () => {
+        const { username } = this.props.match.params;
```

Comment:
`username` and `match` missing in prop validation. 
We can also use destructuring like:
` const { match: { params: {username} } = { params: {} } } = this.props;`

Use destructuring to get items on `this.props`. `this.props` is duplicated in this file. Make the appropriate changes.